packages/launchpad/src/encoding.spec.ts

```diff
@@ -1 +1,50 @@
-describe("encoding", () => {});
+import { sortedJsonStringify } from "./encoding";
+
+describe("encoding", () => {
+  describe("sortedJsonStringify", () => {
+    it("leaves non-objects unchanged", () => {
+      expect(sortedJsonStringify(true)).toEqual(`true`);
+      expect(sortedJsonStringify(false)).toEqual(`false`);
+      expect(sortedJsonStringify("aabbccdd")).toEqual(`"aabbccdd"`);
+      expect(sortedJsonStringify(75)).toEqual(`75`);
+      expect(sortedJsonStringify(null)).toEqual(`null`);
+      expect(sortedJsonStringify([5, 6, 7, 1])).toEqual(`[5,6,7,1]`);
+      expect(sortedJsonStringify([5, ["a", "b"], true, null, 1])).toEqual(`[5,["a","b"],true,null,1]`);
+    });
+
+    it("sorts objects by key", () => {
```

Comment:
I'm wondering if we need to worry about ordering of eg special characters, capital letters etc?

Reply:
So far we never came across such a case for transaction signing. Since all of this is not specified but defined by _do what Go does (what we don't know exactly but let's see)_, I'd avoid adding more logic or tests here.